packages/graphql-server-reactive-core/src/RequestsManager.ts

```diff
@@ -0,0 +1,229 @@
+import { IObservable, Observer, Observable, Subscription } from './Observable';
+import { ReactiveQueryOptions, runQueryReactive } from './runQueryReactive';
+import { ReactiveGraphQLOptions, RGQLExecuteFunction } from './reactiveOptions';
+import { formatError, ExecutionResult } from 'graphql';
+import {
+  RGQL_MSG_ERROR,
+  RGQL_MSG_COMPLETE,
+  RGQL_MSG_DATA,
+  RGQL_MSG_START,
+  RGQL_MSG_STOP,
+  RGQL_MSG_KEEPALIVE,
+  RGQLPacket,
+  RGQLPacketData,
+  RGQLPayloadStart,
+} from './messageTypes';
+
+export class RequestsManager {
```

Comment:
Personally I would prefer to do all the connection handling in the ws package. The reactive core should only provide an observable runQuery function that the ws package can call when appropriate. Kind of like how subscriptions-transport-ws and graphql-subscriptions play together.